includes/otb-utility.php

```diff
@@ -277,6 +277,70 @@ function dc_metadata_to_csv( $dspace_array, $dc_type ) {
 }
 
 /**
+ * @param $number
+ *
+ * @return float|string|void
+ */
+function determineFileSize( $number ) {
+	$result = '';
+	$num    = '';
+
+	//bail if nothing is passed.
+	if ( empty( $number ) ) {
+		return;
+	}
+
+	//if it's a number
+	if ( is_int( $number ) ) {
+		$num = intval( $number );
+	}
+	//only process if it's bigger than zero
+	if ( $num > 0 ) {
+		//return in Megabytes
+		$result = ( $num / 1000000 );
+		//account for the fact that it might be less than 1MB
+		( $result <= 1 ) ? $result = round( $result, 2 ) : $result = intval( $result );
+		$result = "(" . $result . " MB)";
+	}
+
+	return $result;
+}
+
+/**
+ * @param $dspace_array
+ * @param $dc_format
+ *
+ * @return string
+ */
+function dc_bitstream_files( $dspace_array, $dc_format ) {
```

Comment:
I think this function would be better in a namespaced class. It's specific to dspace and not a utility that other classes would find useful. Will consider moving `dc_metadata_to_csv` to the DSpace view for the same reason/consistency.